src/form/datepicker/datepicker.html

```diff
@@ -0,0 +1,3 @@
+<div class="input-group date">
+	<input type="text" class="form-control"/><span class="input-group-addon"><span class="fa fa-calendar"></span></span>
```

Comment:
Also, it think this would be a lot more readable if the html was formatted thusly:

```html
<div class="input-group date">
  <input type="text" class="form-control"/>
  <span class="input-group-addon">
    <span class="fa fa-calendar"></span>
  </span>
</div>
```